modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/GenericConfiguration.java

```diff
@@ -8,7 +8,9 @@
  *
  * @author Pablo Klaschka (@pklaschka), Ludwig Richter (@fussel178)
  * @see TelestionVerticle#getConfig()
+ * @deprecated Use {@link UntypedConfiguration} instead.
```

Comment:
I feel like we could elaborate on that a bit more.

Something along the lines of

> Use {@link UntypedConfiguration} to indicate generic, untyped configuration or {@link NoConfiguration} to indicate no configuration options instead. To build your own typed configuration options, implement {@link TelestionConfiguration}.

Reply:
Done in [`b5b58e3` (#528)](https://github.com/wuespace/telestion-core/pull/528/commits/b5b58e36e03fc73cbf6dd546dfbf3c77f6c78736)